test/Microsoft.AspNetCore.SignalR.Tests.Utils/ServerFixture.cs

```diff
@@ -31,7 +31,7 @@ public ServerFixture()
             var testLog = AssemblyTestLog.ForAssembly(typeof(TStartup).Assembly);
             _logToken = testLog.StartTestLog(null, $"{nameof(ServerFixture<TStartup>)}_{typeof(TStartup).Name}", out _loggerFactory, "ServerFixture");
             _logger = _loggerFactory.CreateLogger<ServerFixture<TStartup>>();
-            Url = "http://localhost:" + GetNextPort();
+            Url = "http://127.0.0.1:" + GetNextPort();
```

Comment:
Put a comment here explaining why (because we want to ensure the RemoteIPAddress value is consistent)